.nycrc

```diff
@@ -0,0 +1,24 @@
+{
+  "extension": [
+    ".js",
+    ".jsx"
+  ],
+  "include": [
+    "src"
+  ],
+  "require": [
+    "babel-register"
+  ],
+  "reporter": [
+    "text",
+    "lcov"
+  ],
+  "all": true,
+  "check-coverage": true,
+  "statements": 89,
```

Comment:
just one final re-confirmation - are these numbers exactly in line with our current coverage? If not, can we bump them to be? :-)